README.md

```diff
@@ -1,29 +1,29 @@
-# Super Rentals
+# super-rentals
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
-
-This is the working repository for the Super Rentals tutorial,
-which you can check out at https://guides.emberjs.com/current/tutorial/ember-cli/.
+This README outlines the details of collaborating on this Ember application.
```

Comment:
Can we restore this block from the original?

Reply:
Done! @sivakumar-kailasam